examples/servicegraph/cmd/server/main.go

```diff
@@ -47,17 +47,45 @@ func (fs justFilesFilesystem) Open(name string) (http.File, error) {
 	return f, nil
 }
 
+func (s *state) addNode(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		w.WriteHeader(http.StatusNotImplemented)
+		_, err := w.Write([]byte("requests of this type not supported at this time"))
+		if err != nil {
+			log.Print(err)
+		}
+		return
+	}
+	nodeName := r.URL.Query().Get("name")
+	if nodeName == "" {
+		w.WriteHeader(http.StatusBadRequest)
+		_, err := w.Write([]byte("missing argument 'name'"))
+		if err != nil {
+			log.Print(err)
+		}
+		return
+	}
+	s.staticGraph.Nodes[nodeName] = struct{}{}
```

Comment:
Right now, nodes don't have any associated metadata.  That can be added later -- at which point this will need to be updated.